dns/dns_updater.go

```diff
@@ -161,6 +167,24 @@ func (u *updater) Update(update controller.IngressUpdate) error {
 	return nil
 }
 
+func (u *updater) filterUnassociated(aRecords []*route53.ResourceRecordSet) []*route53.ResourceRecordSet {
+	managedLBs := make(map[string]bool)
+	for _, dns := range u.schemeToDNS {
+		managedLBs[dns.dnsName] = true
+	}
+	filtered := make([]*route53.ResourceRecordSet, 0)
+	for _, rec := range aRecords {
+		if rec.AliasTarget != nil && rec.AliasTarget.DNSName != nil && managedLBs[*rec.AliasTarget.DNSName] {
+			log.Debugf("Keeping managed A record %s", *rec.Name)
+			filtered = append(filtered, rec)
+		} else {
+			log.Infof("Filtering non-managed A record %s", *rec.Name)
```

Comment:
Can we combine this into a single log line for all filtered records, so it's less spammy.